### PR TITLE
Corrects Power cells, printing and spelling

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
*Im so sorry*
Makes sure to have the cells have their own IDs for sci printing!
Makes sure theirs no grammar problems
Gives the better cells more levels to show that they are better... 
DNM testing it now
See Issue #204

### CODE REVIEW

File: code/modules/research/designs/power_cells.dm [hunk 1]
```diff
@@ -65,24 +65,24 @@
 /datum/design/item/powercell/advance
```
Comment (Hawk-v3) [inline code/modules/research/designs/power_cells.dm]: Your Datums also all conflict, as they're using the same words, as the datums for the APC Cell designs.

File: code/modules/research/designs/power_cells.dm [hunk 2]
```diff
@@ -65,24 +65,24 @@
...
 	name = "advance"
 	build_type = PROTOLATHE
-	id = "advance"
+	id = "advance-device"
```
Comment (Hawk-v3) [inline code/modules/research/designs/power_cells.dm]: Stick to one method for your IDs, don't mix them.

File: code/modules/research/designs/power_cells.dm [hunk 3]
```diff
@@ -65,24 +65,24 @@
...
 	materials = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 50)
 	build_path = /obj/item/weapon/cell/device/weapon
 	category = "Misc"
 	sort_string = "BAABB"
 
 /datum/design/item/powercell/super
 	name = "super"
-	id = "super_cell"
+	id = "super_device"
```
Comment (Hawk-v3) [inline code/modules/research/designs/power_cells.dm]: Stick to one method for your IDs, don't mix them.

**Comment by Hawk-v3** [comment]
Firstly, ignore those conflicts. I'll deal with them when I do the merge. Secondly, you're still mixing use of - and _ in your id= sections.

**Comment by Trilbyspaceclone** [comment]
Local works, but I dont know if the sprites are still in or not...
3rd party tester would be nice...

**Comment by Hawk-v3** [comment]
Status is: You've not overridden the cells any more, but there's a few little bits of tidying up needed. I'll get it done myself.

**Pull request merged**